containers/dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM cmelab/gpuhoomd3conda:latest
```

Comment:
Do we have a gpuhoomd4conda docker file yet? We'll need one now that we updated things to use hoomd4

Reply:
We will have one once the PR in the containers repo is merged.

Reply:
Not yet. I'll throw one together though